site3/website/src/pages/release-notes.md

```diff
@@ -1,6 +1,388 @@
 <!-- markdown-link-check-disable -->
 # Release notes
 
+## 4.16.0
+
+Release 4.16.0 includes multiple bug fixes and some dependencies CVE fixes.
+
+Apache BookKeeper users are encouraged to upgrade to 4.16.0.
+The technical details of this release are summarized below.
+
+### Highlights
+
+#### Configuration
+* BP-60:Change PCBC limitStatsLogging default value to true [PR #3719](https://github.com/apache/bookkeeper/pull/3719)
+
+#### Features
+* [FEATURE] Added api/v1/bookie/cluster_info REST API [PR #3714](https://github.com/apache/bookkeeper/pull/3714)
+* [FEATURE] Added api/v1/bookie/cluster_info REST API [PR #3713](https://github.com/apache/bookkeeper/pull/3713)
+* [feature] Added api/v1/bookie/cluster_info REST API [PR #3710](https://github.com/apache/bookkeeper/pull/3710)
+* Add new api resumeCompaction and suspendCompaction [PR #3509](https://github.com/apache/bookkeeper/pull/3509)
+* BP-58: Change the API for org.apache.bookkeeper.stats.Counter [PR #3501](https://github.com/apache/bookkeeper/pull/3501)
+* Feature: auto recover support repaired not adhering placement ledger [PR #3359](https://github.com/apache/bookkeeper/pull/3359)
+* BP-50: Add reuse journal file feature to support Intel PMem disk [PR #3194](https://github.com/apache/bookkeeper/pull/3194)
+* BP-47 (task1): Add native IO library support [PR #3189](https://github.com/apache/bookkeeper/pull/3189)
+* BP-47 (task4): Aligned native buffer wrapper [PR #3253](https://github.com/apache/bookkeeper/pull/3253)
+* BP-47 (task6): Direct I/O entrylogger support [PR #3263](https://github.com/apache/bookkeeper/pull/3263)
+* BP-47 (task5): Garbage collection support direct IO entrylogger [PR #3256](https://github.com/apache/bookkeeper/pull/3256)
+* BP-47 (task7): DbLedgerStorage add direct entry logger support [PR #3366](https://github.com/apache/bookkeeper/pull/3366)
```

Comment:
Please summarize this as a single line/one feature, list PR numbers or the master issue number. Otherwise it is too noisy and unclear